src/test/java/seedu/programmer/logic/commands/AddLabCommandTest.java

```diff
@@ -18,6 +18,10 @@
 import seedu.programmer.model.student.Lab;
 import seedu.programmer.testutil.LabBuilder;
 
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for
```

Comment:
I think the integration tests are done in a separate test class called `AddCommandIntegrationTest`!

Reply:
okay, makes sense!

Reply:
wait, I think you are confusing this file with AddCommandTest.java

Reply:
Yep, sorry abt that